debug-quick-look.php

```diff
@@ -142,6 +142,7 @@ public function process_debug_type() {
 
 		// Create some basic CSS.
 		$style  = '
+		body#error-page { max-width: 1200px; }
```

Comment:
thinking maybe a percentage instead of an actual pixel value to handle various screen sizes?